autolens/pipeline/phase/abstract/result.py

```diff
@@ -50,7 +53,7 @@ def source_plane_inversion_centres(self) -> grids.Coordinates:
         try:
             return self.most_likely_fit.inversion.brightest_reconstruction_pixel_centre
         except AttributeError:
-            return grids.Coordinates(coordinates=[])
+            pass
```

Comment:
I like to explicitly return None.

If a function can return None then it's best to express that in the type annotation with Optional[grid.Coordinates]